common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -305,7 +330,9 @@ void processMsgQueue(ChannelHandlerContext ctx) {
             log.trace("[{}][{}] Postpone processing msg due to device is not connected. Msg queue size is {}", sessionId, deviceSessionCtx.getDeviceId(), deviceSessionCtx.getMsgQueueSize());
             return;
         }
-        deviceSessionCtx.tryProcessQueuedMsgs(msg -> processRegularSessionMsg(ctx, msg));
+        deviceSessionCtx.tryProcessQueuedMsgs(msg -> {
+            processRegularSessionMsg(ctx, msg);
+        });
```

Comment:
consider to rollback this, since you anyway need to fix typo: **sendResponseForAdapterError**